comet-accumulo/src/main/java/org/renci/comet/CometInitializer.java

```diff
@@ -12,7 +12,7 @@
 @EnableSwagger2
 @ComponentScan(basePackages = { "org.renci.comet", "org.renci.comet.api" })
 public class CometInitializer implements CommandLineRunner {
-	public static final String COMET_VERSION = "1.0.0";
+	public static final String COMET_VERSION = "1.0.5";
 	public static final String COMET_API_VERSION = "1.0.0";
 	public static final String COMET_SCHEMA_VERSION = "1.0.0";
```

Comment:
Schema version should be increased too, since we are adding new fields.

Reply:
good catch, thanks!

Reply:
Just pushed a new commit to update this